### PR TITLE
Add FPS to video file name

### LINKED ISSUE #106: Add frame rate to video file name
I suggest this convention
```python
"{hostname}_FR{framerate}_{datetime}"
```
instead of the existing one
```python
"{hostname}_{datetime}"
```

### PR DESCRIPTION
**Pull request by randy-seng**
fixes #106

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -170,3 +170,5 @@ Thumbs.db
 # Local History for Visual Studio Code
 .history/
 
+# Vim
+.vim/coc-settings.json
```
Comment (frunika) [inline .gitignore:174]: no comment :D

File: OTCamera/helpers/name.py [hunk 2]
```diff
@@ -87,22 +92,50 @@ def preview() -> str:
...
+
+    try:
+        date_time = dt.strptime(datetime_str, "%Y-%m-%d_%H-%M-%S")
+    except ValueError:
+        return None
+
+    return date_time
+
+
+def get_fps_from_filename(filename: str) -> int:
```
Comment (frunika) [inline OTCamera/helpers/name.py:125]: Shouldn't we refactor methods only used by tests to the test itself?

**Review by frunika — COMMENTED** [review]
see inline comments

File: OTCamera/helpers/name.py (hunk 2 above)
Comment (randy-seng) [reply OTCamera/helpers/name.py:125]: Ah yes I added the method and its respective unit tests to the codebase in case we want to use it in the future.

**Review by frunika — APPROVED** [review]

**Pull request merged**